devicetypes/smartthings/smartsense-moisture.src/smartsense-moisture.groovy

```diff
@@ -21,7 +21,8 @@ metadata {
 
 		fingerprint deviceId: "0x2001", inClusters: "0x30,0x9C,0x9D,0x85,0x80,0x72,0x31,0x84,0x86"
 		fingerprint deviceId: "0x2101", inClusters: "0x71,0x70,0x85,0x80,0x72,0x31,0x84,0x86"
-		fingerprint mfr:"0084", prod:"0063", model:"010C", deviceJoinName: "FortrezZ Moisture Sensor"
+		fingerprint mfr:"0084", prod:"0063", model:"010C"
+		fingerprint mfr:"0084", prod:"0053", model:"0216", deviceJoinName: "FortrezZ Moisture Sensor"
 	}
```

Comment:
Wasn't this device branded as a SmartSense Moisture sensor?

Reply:
Yeah, I was about to comment on that, but it should take up that name from the DTH name. Or, at least, "SmartSense Moisture". Maybe it should have been "SmartSense Moisture Sensor"?

Reply:
I had samples brought to my desk and saw it with my own eyes which is which.